spec/models/location_spec.rb

```diff
@@ -1,5 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Location, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'attribute validations' do
+    it 'does not validate when name is not defined' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.